octohatrack/__init__.py

```diff
@@ -17,13 +17,17 @@ def main():
 
   repo_name = args.repo_name
 
-  if not repo_exists(repo_name): 
-    print("Repo does not exist: %s" % repo_name)
+  try:
```

Comment:
yes, that's why i put it in :) but its not instantly obvious if this codebase does all the API requests in these 3 calls, and then is (only) doing output of data, or if there are other API requests hidden in the rendering code.